Assets/Scripts/CoinAmountText.cs

```diff
@@ -0,0 +1,24 @@
+﻿using UnityEngine;
+using TMPro;
+
+public class CoinAmountText : MonoBehaviour
+{
+    [SerializeField] private TMP_Text _amountText;
+
+    [SerializeField] private PlayerController _player;
+
+    private void OnEnable()
+    {
+        _player.GetCurrentCoinAmount += SetCoinAmount;
```

Comment:
https://trello.com/c/CsOZfMhi/1-%D0%BD%D0%B0%D0%B7%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9-%D0%BE%D0%B1%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D1%87%D0%B8%D0%BA%D0%BE%D0%B2